phpstan.neon.dist

```diff
@@ -0,0 +1,9 @@
+parameters:
+   level: 1
```

Comment:
This is way too low compared to on what level Psalm ran.

Reply:
Yeah :( I will try to increase it slowly. Psalm was running on high level because of carefully crafted suppression rules, somebody has to put in the work to make them for phpstan. I've already run into https://github.com/phpstan/phpstan/issues/12414 (this is the kind of things baselines hide)

I'll make it to lvl2 today. Let me know if you would be interested in help to update it to some other lvl.

Reply:
I'll have a look, but I won't be able to start right away. A temporary baseline would've been good in order to not lose coverage and track the progress.